.gitignore

```
@@ -76,3 +76,6 @@ typings/
 .fusebox/
 includes/.DS_Store
 volkswagen/.DS_Store
+prajwal-vw-header/includes/.DS_Store
+prajwal-vw-header/.DS_Store
+.DS_Store
```

@praxoc 

1. Instead of adding multiple ignore rules try using the [Git ignore pattern ](https://www.atlassian.com/git/tutorials/saving-changes/gitignore)to ignore every type of file.

```suggestion
**/.DS_Store
```

2. Please remove the unused boilerplate `.gitignore` rules. This file could be as simple as this:
```git
node_modules/
**/.DS_Store
```